common/amundsen_common/models/generation_code.py

```diff
@@ -6,13 +6,14 @@
 
 
 @attr.s(auto_attribs=True, kw_only=True)
-class Query:
-    name: Optional[str]
+class GenerationCode:
```

Comment:
I personally would put this into feature.py and call it FeatureGenerationCode. In databuilder I did not make it anyhow general and it's namespaced under features.